easybuild/easyconfigs/b/Boost.Python/Boost.Python-1.85.0-GCC-13.3.0.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = 'EB_Boost'
+
+name = 'Boost.Python'
+version = '1.85.0'
+
+homepage = 'https://boostorg.github.io/python'
+description = """Boost.Python is a C++ library which enables seamless interoperability between C++
+ and the Python programming language."""
+
+toolchain = {'name': 'GCC', 'version': '13.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://boostorg.jfrog.io/artifactory/main/release/%(version)s/source/']
+sources = ['boost_1_85_0.tar.gz']
```

Comment:
heh, the sources are not downloaded from this location, it's just a html file..

probably better to get them from here:
https://github.com/boostorg/boost

there is also v1.87.0 available, is there a reason not to use that one?

Reply:
@smoors I just copied it from @jpecar 's PR #21864
As far as I know Boost.Python and Boost.Python-Numpy need to be the same version as Boost. We have 1.85.0 for GCC 13.3.0
I'll fix the source_urls.

Is there some parameter use-same-source-as to use the tar.gz of Boost-1.85.0-GCC-13.3.0 ?

Reply:
done

Reply:
>Is there some parameter use-same-source-as to use the tar.gz of Boost-1.85.0-GCC-13.3.0 ?

not that i know of